pensjon-brevbaker-api-model/src/main/kotlin/no/nav/pensjon/brev/api/model/maler/EndringIOpptjeningAuto.kt

```diff
@@ -0,0 +1,80 @@
+package no.nav.pensjon.brev.api.model.maler
+
+import no.nav.pensjon.brev.api.model.Kroner
+import no.nav.pensjon.brev.api.model.Sivilstand
+import no.nav.pensjon.brev.api.model.vedlegg.MaanedligUfoeretrygdFoerSkattDto
+import no.nav.pensjon.brev.api.model.vedlegg.OpplysningerBruktIBeregningUTDto
+import no.nav.pensjon.brev.api.model.vedlegg.OrienteringOmRettigheterUfoereDto
+import java.time.LocalDate
+
+@Suppress("unused")
+data class EndringIOpptjeningAutoDto(
+    val brukerBorInorge: Boolean,
+    val fellesbarn1: Fellesbarn1?,
```

Comment:
Vi skal ikke ende variabel-navn eller klassenavn med 1. Klassen kan godt hete EndringIOpptjeningAutoDtoFellesbarn for å unngå konflikt på grunn av en feil vi bør fikse snart, men variabelnavn som `fellesbarn1` kan hete det den vil så lenge den ikke er i konflikt med noe i samme klassen. Da blir dette `val fellesbarn: EndringIOpptjeningAutoDtoFellesbarn?`